### PR TITLE
[39_13] update doc in Chap. 3.8

### PR DESCRIPTION
**Pull request by JackYansongLi**
Solve the wrong reference link in Chapter 3.8

### CODE REVIEW

File: TeXmacs/doc/main/Mogan.zh.tmu [hunk 1]
```diff
@@ -553,7 +553,7 @@
 
   <section|精通键盘><label|sec:master_keyboard><rsub|>
 
-  本节主讲快捷键，相关内容除本节外还可以参考<smart-ref|sec:config_keyboard>。
+  本节主讲快捷键，相关内容除本节外还可以参考节-<reference|sec:convention>和<smart-ref|sec:config_keyboard>。
```
Comment (da-liii) [inline TeXmacs/doc/main/Mogan.zh.tmu:556]: 这里应该使用 smart-ref

File: TeXmacs/doc/main/Mogan.en.tmu [hunk 2]
```diff
@@ -681,7 +681,7 @@
 
   <section|Mastering the keyboard>
 
-  We recall that the section on <hlink|general conventions|../../start/man-conventions.en.tm> contains explanations on the way keyboard shortcuts are printed in this manual. It may also be useful to take a look at the section on <hlink|keyboard configuration|../../config/man-config-keyboard.en.tm>.
+  We recall that the Section <reference|sec:general-convention> on <with|font-shape|italic|general convention> contains explanations on the way keyboard shortcuts are printed in this manual. It may also be useful to take a look at <with|font-shape|italic|keyboard configuration> mention in Section <reference|sec:kbd-config>.
```
Comment (da-liii) [inline TeXmacs/doc/main/Mogan.en.tmu:684]: 使用 smart-ref

Comment (JackYansongLi) [reply TeXmacs/doc/main/Mogan.en.tmu]: done

File: TeXmacs/doc/main/Mogan.zh.tmu (hunk 1 above)
Comment (JackYansongLi) [reply TeXmacs/doc/main/Mogan.zh.tmu]: done

File: TeXmacs/doc/main/Mogan.en.tmu [hunk 3]
```diff
@@ -681,7 +681,7 @@
 
   <section|Mastering the keyboard>
 
-  We recall that the section on <hlink|general conventions|../../start/man-conventions.en.tm> contains explanations on the way keyboard shortcuts are printed in this manual. It may also be useful to take a look at the section on <hlink|keyboard configuration|../../config/man-config-keyboard.en.tm>.
+  We recall that the <smart-ref|sec:general-convention> on <with|font-shape|italic|general convention> contains explanations on the way keyboard shortcuts are printed in this manual. It may also be useful to take a look at <with|font-shape|italic|keyboard configuration> mention in <smart-ref|sec:kbd-config>.
```
Comment (da-liii) [inline TeXmacs/doc/main/Mogan.en.tmu:684]: using the semantic tag `em` is better than the italic font shape

**Review by da-liii — APPROVED** [review]
LGTM

File: TeXmacs/doc/main/Mogan.en.tmu (hunk 3 above)
Comment (JackYansongLi) [reply TeXmacs/doc/main/Mogan.en.tmu]: Good idea! done!

**Pull request merged**